src/company_cmd.cpp

```diff
@@ -870,12 +873,18 @@ CommandCost CmdCompanyCtrl(TileIndex tile, DoCommandFlag flags, uint32 p1, uint3
 			break;
 		}
 
-		case CCA_NEW_AI: { // Make a new AI company
+		case CCA_NEW_AI: { // Make new AI companies
+			uint num_to_start = GB(p2, 0, 4);
+			if (num_to_start == 0) return CMD_ERROR;
+
 			if (!(flags & DC_EXEC)) return CommandCost();
 
 			if (company_id != INVALID_COMPANY && (company_id >= MAX_COMPANIES || Company::IsValidID(company_id))) return CMD_ERROR;
```

Comment:
Should probably be moved above the DC_EXEC check, and modified to enforce INVALID_COMPANY if multiple num_to_start > 1